src/form-sections/amount.ts

```diff
@@ -60,22 +61,13 @@ export class MGCEditPlanAmount extends LitElement {
       <section>
         <donation-form-section badgemode="hidebadge" headline="Amount">
           ${!this.currentlyEditing
-            ? html` <p class="current-amount">
-                  USD $${this.plan?.amount}
-                  <ia-mgc-update-status .status=${this.updateStatus}
-                    >${this.updateMessage}</ia-mgc-update-status
-                  >
-                </p>
-                <ia-button
-                  id="edit-amount"
-                  class="ia-button link"
-                  .clickHandler=${() => {
-                    this.currentlyEditing = true;
-                    this.clearStatusMessaging();
-                  }}
-                >
-                  Edit...
-                </ia-button>`
+            ? html`<ia-mgc-form-section-info
+                @editingRequested=${() => {
+                  this.currentlyEditing = true;
+                  this.clearStatusMessaging();
+                }}
+                ><span>USD $${this.plan?.amount}</span>
```

Comment:
QUESTION: If `this.plan` is null, is it correct that this span will display `USD $` with nothing else?